pkg/nsx/services/common/types.go

```diff
@@ -69,3 +69,9 @@ var (
 	String = pointy.String // address of string
 	Int64  = pointy.Int64  // address of int64
 )
+
+type OrgProjectVPC struct {
```

Comment:
The struct name is a little weird to me..

Reply:
It consists of org, project, and vpc, which is a common struct used often.